rebar.config

```diff
@@ -2,6 +2,9 @@
     debug_info,
     {platform_define, "^19", 'OTP_19'}
 ]}.
+
+{project_plugins, [rebar3_ex_doc]}.
```

Comment:
Alternatively, you can just rely on this being in maintainers global config. I always forget about that, so opt to put it in project_plugins.